### PR TITLE
Html css popup

### PR DESCRIPTION
**Pull request by Serge044**
### HTML-CSS-Popup

[Demo](https://serge044.github.io/HTML-CSS-Popup/) |
[Code base](https://github.com/Serge044/HTML-CSS-Popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/Serge044/HTML-CSS-Popup/index.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+                        <a href="#Images" class="main__nav-buttons images"><span
+                                class="main__nav-buttons__text">Images</span></a>
+                    </li>
+                    <li>
+                        <input class="popup_btn" type="checkbox">
+                        <div class="popup__menu">
+                            <ul class="popup__menu__main">
+                                <li class="popup__menu__item">
+                                    <a href="#Calendar">
+                                        <img src="images/kottans-calendar.png" alt="Calendar" class="popup__icons">
```
Comment (HannaSyn) [inline submissions/Serge044/HTML-CSS-Popup/index.html:31]: For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)

File: submissions/Serge044/HTML-CSS-Popup/index.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <nav class="nav__menu">
+                <ul class="ul__items">
+                    <li>
+                        <a href="#Mail" class="main__nav-buttons mail"><span
```
Comment (HannaSyn) [inline submissions/Serge044/HTML-CSS-Popup/index.html:18]: Please, read more about [BEM](https://en.bem.info/methodology/)
I don't see class name of block `main__nav`

**Review by HannaSyn — CHANGES_REQUESTED** [review]
Wow, you did a great work, I almost don't have comments, well done! 
See few comments to make it little better

**Comment by Serge044** [comment]
Thanks for your review! I made the changes and it should be better now.

**Pull request merged**